src/v2/Components/v2/ArtworkFilter/ArtworkFilterMobileActionSheet.tsx

```diff
@@ -40,7 +53,7 @@ export const ArtworkFilterMobileActionSheet: SFC<{
         </Button>
       </Header>
 
-      <Content>
+      <Content ref={contentRef as any}>
```

Comment:
Should this be typed as `HTMLDivElement` to match the declaration on L16?

Reply:
I wish — it shouldn't being cast at all. Something is wrong with our typings for styled-components. (I think it's just a question of upgrading)